.vscode/settings.json

```diff
@@ -9,7 +9,7 @@
     },
     "eslint.enable": false,
     "flow.enabled": false,
-
+    "editor.formatOnSave": true,
```

Comment:
as a compulsive saver (i occasionally try to `cmd-s` in real life), does this noticeably slow down the editor or hang?